### PR TITLE
Wildfly10

### PR DESCRIPTION
**Pull request by lucasponce**
Wildfly10 work.
Some re-organization on profiles to use same itest name for integration.

### CODE REVIEW

File: hawkular-alerts-rest-tests/pom.xml [hunk 1]
```diff
@@ -165,8 +165,9 @@
 
   <profiles>
 
+    <!-- This profile requires a running server of hawkular or hawkular-alerts/standalone -->
     <profile>
-      <id>rest-only</id>
+      <id>itest-tests</id>
```
Comment (jshaughn) [inline hawkular-alerts-rest-tests/pom.xml]: Please change to -Pitest-external to make more verbose that this option requires the external server to be running.

File: hawkular-alerts-rest-tests/pom.xml [hunk 2]
```diff
@@ -353,7 +437,14 @@
...
                   <includes>
-                    <include>**/*ITest*</include>
+                    <include>**/ActionsITest*</include>
+                    <include>**/AlertsITest*</include>
+                    <include>**/ConditionsITest*</include>
+                    <include>**/DampeningITest*</include>
+                    <include>**/EventsITest*</include>
+                    <include>**/EventsLifecycleITest*</include>
+                    <include>**/LifecycleITest*</include>
+                    <include>**/TriggersITest*</include>
```
Comment (jshaughn) [inline hawkular-alerts-rest-tests/pom.xml]: Why did you do this?  It means that if we add a new Itest we'll need to remember to add it here.

**Comment by jshaughn** [comment]
The PR looks good.  Please update that profile name unless you have an issue with the proposal.  I am not sure we should merge this until we have a hawkular that is also running WF10.

File: hawkular-alerts-rest-tests/pom.xml (hunk 1 above)
Comment (lucasponce) [inline hawkular-alerts-rest-tests/pom.xml]: +1

File: hawkular-alerts-rest-tests/pom.xml (hunk 2 above)
Comment (lucasponce) [inline hawkular-alerts-rest-tests/pom.xml]: For some reason, the exclusion rule (which is more simple) it doesn't work well on this plugin.
I'm going to re-check but that was the main reason.
Putting a excludes rules didn't work.

**Comment by lucasponce** [comment]
Thanks for the review.
As commented, I'm going to prepare a commit for the feedback and I won't merge on master, instead, I will create a Wildfly10 branch on hawkular-alerts.
Better to maintain both until hawkular/master is fully moved to Wildfly10.

**Pull request closed without merging**